src/lib/layer-manager.js

```diff
@@ -594,6 +594,10 @@ export default class LayerManager {
    *                        {Object} srcEvent:             native JS Event object
    */
   _onPointerMove(event) {
+    if (event.isDown) {
+      // Is dragging
```

Comment:
this comment means we should not fire pointermove event when mouse is down because pointermove leads to pickLayer getting called. Yes, we probably want to explain a bit more in the comments, or considering put a link to this PR